.github/pull_request_template.md

```diff
@@ -1,12 +1,20 @@
 ## I'm submitting a ...
 
-- [ ] bug report
-- [ ] feature request
+- bug report
+- feature request
 
 ## What is the current behavior?
 
-## If the current behavior is a bug, please provide the steps to reproduce and if possible a minimal demo of the problem
-
 ## What is the expected behavior?
 
 ## What is the motivation / use case for changing the behavior?
+
+## Checklist
+
+- [ ] code is easy to understand
+- [ ] tests are present (if necessary)
+- [ ] no commented out code
+- [ ] no debbugging code
+- [ ] errors are handled
+- [ ] passed linter
```

Comment:
`nit` maybe we already run `ruff` as a workflow automatically, so we can be sure that it's done?